src/repository/friend.repository.ts

```diff
@@ -10,3 +10,16 @@ export const AddFriend = async (userId: string, targetUserId: string) => {
     },
   });
 };
+export const CancelFriendRequest = async (
```

Comment:
함수 이름은 대문자로 시작하지 않기로 해서 아래와 같이 수정하는게 나아보이네요
```suggestion
export const deleteFriendRequest = async (
```